lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -628,6 +628,11 @@ llvm::DIType *CGDebugInfo::CreateType(const TypeVariableType *TvT) {
   return nullptr;
 }
 
+llvm::DIType *CGDebugInfo::CreateType(const ExistentialType *Ty) {
+  // TODO: is this correct?
+  return nullptr;
+}
```

Comment:
I believe you should call CreateType on the body of the existential.  Please file an issue to "try out debugging of code with existential types."

Reply:
Done. See https://github.com/microsoft/checkedc-clang/issues/661#issuecomment-533344812